projectq/backends/_resource.py

```diff
@@ -35,6 +38,9 @@ class ResourceCounter(BasicEngine):
             where ctrl_cnt is the number of control qubits.
         max_width (int): Maximal width (=max. number of active qubits at any
             given point).
+    Properties:
+        depth (int): Circuit depth. It is the longest path in the directed
```

Comment:
I wouldn't call this circuit depth but "depth of DAG".

Reply:
👍 it's now called `depth_of_dag`